app/views/global/_header.html.erb

```diff
@@ -4,10 +4,12 @@
       <h1>Global Database<br>on Protected Area Management Effectiveness</h1>
       <span class="header__divider"></span>
       <p><strong>The Global Database on Protected Area Management Effectiveness</strong> (<%= link_to 'GD-PAME', 'https://protectedplanet.net/c/protected-areas-management-effectiveness-pame', title: 'View information about Protected Areas Management Effectiveness', target: '_blank' %>) is the most comprehensive global database of management effectiveness assessments for protected areas (PAME). It indicates if a protected area documented in the World Database on Protected Areas (<%= link_to 'WDPA', 'https://protectedplanet.net/', title: 'View the World Database on Protected Areas website', target: '_blank' %>) has been assessed. The GD-PAME is a searchable database that includes assessments submitted by a wide range of governmental and non-governmental organizations to UNEP-WCMC, and is updated on a monthly basis.</p>
+      <p>Last update: <%= @updated_at %></p>
```

Comment:
'Last update:' is bold in the designs, so can <strong> tags be popped around that bit (but not the date)? Otherwise looks good!

Reply:
Done!